### ISSUE TITLE
Question: Juno and OSX buttons?

### ISSUE THREAD

**Issue by clintwood** [surajmandalcell/elementary-x#10]
Love this theme, however, is it able to correctly theme the OSX style buttons on all apps including _pantheon_ apps.  From what I'm seeing the developers (specifically the designers) seem to have prevented this?

EDIT: I'm specifically referring to Juno Beta- not Loki

**Comment by surajmandalcell** [comment]
I've tested this in juno(specially juno) and fixed most problems so there shouldn't be any problems. If you have any please provide screenshots and ill fix them possible using stylesheet.

**Comment by surajmandalcell** [comment]
And yes it does perfectly theme osx style buttons on pantheon apps as well as gtk apps!

**Comment by clintwood** [comment]
Wow thank you - I'have a clean install of Juno and followed your [installation](https://github.com/surajmandalcell/elementary-x#installation) guide and not all apps have their buttons on the left OSX style.

Specifically the problem I was having is setting the buttons to be OSX style - i.e. they were split with close (red) on the left and maximize (green) on the right and no minimize (yellow) at all.

However, although I had read that Elementary Tweaks did not work on Juno I installed it using this ppa `ppa:philip.scott/elementary-tweaks` and only when I selected the theme (by changing back to `elementary` then to `elementary-x` again) via Appearance > GTK+ did it work.

So I assume there is some other setting besides what your install guide shows to get this to work!

I did try `gsettings set org.pantheon.desktop.gala.appearance button-layout 'close,minimize,maximize'` but that only moved the close button to the left.

Thanks for a great theme - feel free to close this issue.
